src/mixins/form-text.js

```diff
@@ -158,26 +166,31 @@ export default {
       if (evt.target.composing) {
         return
       }
-      const formatted = this.getFormatted(evt.target.value, evt)
+      const value = evt.target.value
+      const formattedValue = this.formatValue(value, evt)
       // Exit when the `formatter` function strictly returned `false`
       // or prevented the input event
-      if (formatted === false || evt.defaultPrevented) {
-        /* istanbul ignore next */
+      /* istanbul ignore next */
+      if (formattedValue === false || evt.defaultPrevented) {
         evt.preventDefault()
         return
       }
-      this.updateValue(formatted)
-      this.$emit('change', formatted)
+      this.localValue = formattedValue
+      this.updateValue(formattedValue)
```

Comment:
@jackmu95 do you think we should use the `force` option of `updateValue` on change?

Reply:
Added force true for the change event, to better mimic Vue's lazy modifier